File: modul 4/PRAK403-2410817320004-AmandhaCitraMustika.py
awal, akhir = map(int, input().split())
if awal<=akhir:
    i, j = awal, akhir
    while i<j :
        print(f"{i} {j}", end=" ")
        if i<j :
            print("-", end=" ")
        i+=1
        j-=1
    while i<=akhir:
        print(f" - {i} {j}", end=" ")
        i+=1
        j-=1
else :
    i,j = awal,akhir
    while i>=j:
        print(f"{i} {j}", end=" ")
        if i>j :
            print("-", end=" ")
        i-=1
        j+=1
    while i>=akhir:
        print(f" - {i} {j}", end=" ")
        i-=1
        j+=1

File: modul 4/PRAK403-2410817320004-AmandhaCitraMustika.c
#include <stdio.h>
int main(){
int awal, akhir, i,j;
scanf("%d %d", &awal, &akhir);
for (int i=awal, j=akhir; i<=akhir && j>=awal; i++,j--){
    printf("%d %d", i,j);
    if(i<akhir || j>awal) {printf(" - ");}
}
for(int i=awal, j=akhir; i>=akhir && j<=awal; i--, j++){
    printf("%d %d", i,j);
    if(i>akhir || j<awal) {printf(" - ");}
}
    return 0;
}